client/src/Components/Login.js

```diff
@@ -0,0 +1,149 @@
+import Input from "./Input";
+import Button from "./Button";
+import emailIcon from "../images/email.svg";
+import passwordIcon from "../images/password.svg";
+import googleIcon from "../images/google.svg";
+import visibilityIcon from "../images/visibility_off.svg";
+
+const Login = () => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const user = useSelector(state => state.auth);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    dispatch(login(email, password));
+  };
+
+  if (user.isLoggedIn) {
+    return <Redirect to="/" />;
+  }
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit} className={classes.form}>
+        <FormControl>
+          <label className={classes.label}>
+            <div className={classes.helperText}>
+              <img src={emailIcon} alt="email" className={classes.icon} />
+              Email or Username
+            </div>
+            <Input
+              name="email"
+              type="email"
+              value={email}
+              onChange={e => setEmail(e.target.value)}
+            />
+          </label>
+          <label className={classes.label}>
+            <div className={classes.helperText}>
+              <img src={passwordIcon} alt="password" className={classes.icon} />
+              Password
+            </div>
+            <Input
+              name="password"
+              type="password"
+              value={password}
+              onChange={e => setPassword(e.target.value)}
+              inputProps={{
+                endAdornment: (
+                  <img
+                    src={visibilityIcon}
+                    alt="visibility_off"
+                    className={classes.visibilityIcon}
+                  />
+                ),
+              }}
+            />
+          </label>
+          <h4 className={classes.font} style={{ textAlign: "end" }}>
+            Forgot Password?
+          </h4>
+          <Button>Submit</Button>
+        </FormControl>
+      </form>
+      <div className={classes.login}>
+        <h4
+          className={classes.font}
+          style={{ textAlign: "center", margin: "0px" }}
+        >
+          or
+        </h4>
+        <div className={classes.loginService}>
+          <img src={googleIcon} alt="google" className={classes.googleIcon} />
+          <h4 className={classes.font} style={{ fontWeight: "800" }}>
+            Continue with Google
+          </h4>
+        </div>
+        <h4 className={classes.font} style={{ color: "#717171" }}>
+          Dont' have an account?
+          <Link to="/signup" className={classes.signup}>
+            <span className={classes.font} style={{ fontWeight: "800" }}>
+              {" "}
+              Sign up
+            </span>
+          </Link>
+        </h4>
+      </div>
+    </div>
+  );
+};
+
+const useStyles = makeStyles(() => ({
```

Comment:
To follow convention, let's move the useStyles object to the top of the component.